include/swift/AST/USRGeneration.h

```diff
@@ -44,7 +44,8 @@ bool printDeclTypeUSR(const ValueDecl *D, raw_ostream &OS);
 
 /// Prints out the USR for the given ValueDecl.
 /// \returns true if it failed, false on success.
-bool printValueDeclUSR(const ValueDecl *D, raw_ostream &OS);
+bool printValueDeclUSR(const ValueDecl *D, raw_ostream &OS,
```

Comment:
Could you add the doc comment for the parameter here too?

Reply:
Done.